### PR TITLE
Add apivault in made-in-italy

### PR DESCRIPTION
**Pull request by FlavioAdamo**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> I have added apivault.dev to the made-in-italy collection, our website is currently helping thousand of developers around the world, and all the maintainer are italian!

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/made-in-italy/index.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ items:
  - Schrodinger-Hat/ImageGoNord
  - stoplightio/prism
  - ercole-io/ercole
+ - Exifly/ApiVault
+
```
Comment (ahpook) [inline collections/made-in-italy/index.md:21]: Please trim this extra newline
```suggestion
```

**Review by ahpook — CHANGES_REQUESTED** [review]
Rogue newline, otherwise LGTM

**Comment by FlavioAdamo** [comment]
> Rogue newline, otherwise LGTM

done! 👍🏻

**Review by ahpook — APPROVED** [review]
Thank you!

**Pull request merged**